src/components/AddBeer.js

```diff
@@ -0,0 +1,118 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+import Header from './Header';
+
+export class AddBeer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: '',
+      tagline: '',
+      description: '',
+      first_brewed: '',
+      brewers_tips: '',
+      attenuation_level: 0,
+      contributed_by: '',
+    };
+  }
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+    axios
+      .post('https://ih-beers-api2.herokuapp.com/beers/new', this.state)
+      .then((response) => {
+        console.log(response.data);
+      })
+      .catch((err) => console.log('Error when submiting a new beer', err));
+  };
+
+  handleChange = (event) => {
```

Comment:
que seria attenuation_level?

```
handleChange = ({target}) => {
  const hasToParseValue = target.name === "attenuation_level"
const value = hasToParseValue ? parseInt(target.value) : target.value
 this.setState(state => ({
   ...state,
   [target.name]: value 
}))
```

si necesitas hacer uso del currentState para actualizar tu estado como en este ejemplo hacelo con callback.
el valor the this.state al estar referido a la palabra this puede no ser lo que estas buscando, con el callback te aseguras de que asi sea.
}